src/components/Filters.module.sass

```diff
@@ -0,0 +1,6 @@
+#filterContainer
```

Comment:
Why do you need this an ID and not a class?

Reply:
So I won't have to use `!important` and to make the filter designed with flex